apiary.apib

```diff
@@ -3574,6 +3572,14 @@ All projects using the backend must be deleted and purged before it can be delet
 + owner: keboola (required) - associated AWS account owner
 + database: MyDataWarehouse - required only for Synapse
 + useSynapseManagedIdentity: true - optional for Synapse
++ useDynamicBackends: false (boolean) - Only for backends supporting dynamic sizing (Snowflake). When enabled, new projects get dynamic backends assigned automatically.
```

Comment:
Mozno by som tam doplnil ze je to stale beta feature pripadne mozno niekam do desc. ze co to vlastne to je ten dynamic backend skoda ze ktomu nemame dokumentaciu tak mozno len toto https://community.keboola.com/post/dynamic-backend-size-for-snowflake-jobs-public-beta-618a66d64930870fec54fb45

Reply:
To mi přijde irelevantní. To má bejt v dokumentaci. Navíc je to superuser věc...

Reply:
A jo vlastne okej :) bol to len napad ;)

Reply:
Jakože tohle je API docs. Ne "how to keboola". Když nevíš, je to optional, tak to nevyplňuj.

Reply:
Jo, dobrej point, o tom žádná.